shared/docs/PROTOCOL_STATE_HASH.md

```diff
@@ -107,13 +107,13 @@ graph TD
     D[Did I get any prepareQCs?] --> |No| Z
 
     E[Am I ahead of highPrepareQC?] --> |Yes| G
-    E[Am I ahead of highPrepareQC?] --> |No| Z
+    E[Am I ahead of prepareQC?] --> |No| Z
```

Comment:
Why was this changed on line 110 but not line 109? I feel like it should be done either everywhere or nowehere.

Reply:
that is for sure right. Probably some ctrl z mistake, that went in unnoticed, thanks for catching.